components/automate-backend-haproxy/habitat/config/haproxy.conf

```diff
@@ -20,10 +20,10 @@ listen postgresql
     default-server inter 3s fall 3 rise 2 on-marked-down shutdown-sessions
 {{~#if bind.database}}
   {{~#eachAlive bind.database.members as |dbmember|}}
-    server {{dbmember.sys.ip}} {{dbmember.sys.ip}}:{{dbmember.cfg.port}} maxconn {{cfg.server.maxconn}} check port "${PGLEADERCHK_PORT}"
+    server {{dbmember.sys.ip}} {{dbmember.sys.ip}}:{{dbmember.cfg.port}} maxconn {{cfg.server.maxconn}} check port 6432
   {{~/eachAlive}}
 {{~else}}
-    server {{sys.ip}} {{sys.ip}}:{{cfg.pg.port}} maxconn {{cfg.server.maxconn}} check port "${PGLEADERCHK_PORT}"
+    server {{sys.ip}} {{sys.ip}}:{{cfg.pg.port}} maxconn {{cfg.server.maxconn}} check port 6432
```

Comment:
we can use this 
6432 => {{cfg.pgleaderchk.port}}

Reply:
Done.